components/slack/sources/common/base.mjs

```diff
@@ -1,21 +1,77 @@
 import slack from "../../slack.app.mjs";
+import {
+  NAME_CACHE_MAX_SIZE, NAME_CACHE_TIMEOUT,
+} from "./constants.mjs";
 
 export default {
   props: {
     slack,
-    nameCache: "$.service.db",
+    db: "$.service.db",
   },
   methods: {
-    async maybeCached(key, refreshVal, timeoutMs = 3600000) {
-      let record = this.nameCache.get(key);
+    cleanCache(cacheObj) {
+      console.log("Initiating cache check-up routine...");
```

Comment:
```suggestion
      console.log("Initiating cache check-up...");
```

Reply:
Changed 'check-up routine' to 'check-up' to make it concise (Google's Developer docs style guide: Be concise)